Moda.Web/src/moda.web.reactclient/src/services/queries/organization-queries.ts

```diff
@@ -0,0 +1,32 @@
+import { useQuery } from 'react-query'
+import { QK } from './query-keys'
+import { getTeamsClient, getTeamsOfTeamsClient } from '../clients'
+
+// TEAMS - RISKS
+export const useGetTeamRisks = (
+  id: string,
+  includeClosed: boolean = false,
+  enabled: boolean = true,
+) => {
+  return useQuery({
+    queryKey: [QK.TEAM_RISKS, id, { includeClosed: includeClosed }],
```

Comment:
minor, but this could be simplified to just: queryKey: [QK.TEAM_RISKS, id, includeClosed ], the named object property MIGHT cause issues with the query key. We found best to just use primitives.

Reply:
roger that